editor/gui/editor_toaster.cpp

```diff
@@ -371,6 +371,20 @@ Control *EditorToaster::popup(Control *p_control, Severity p_severity, double p_
 	p_control->set_h_size_flags(SIZE_EXPAND_FILL);
 	hbox_container->add_child(p_control);
 
+	// Copy button.
+	if (p_time > 0.0) {
```

Comment:
This condition is the same as the one below, so you can create both buttons in one block. Also the comments are unnecessary.

Reply:
Done, ty.